src/kernel/mconfig.ml

```diff
@@ -118,6 +118,8 @@ type merlin = {
 
   failures    : string list;
 
+  assocsuffixes : (string * string) list
```

Comment:
I'd be in favor of a more explicit name like `extension_to_reader` (or `suffix_to_reader` if we want to keep the `suffix` term).

Reply:
ok thanks - will update